src/T2DMap.cpp

```diff
@@ -3002,11 +3004,11 @@ void T2DMap::slot_customLineProperties()
             }
 
             QStringList lineStyles;
-            lineStyles << "solid line"
-                        << "dot line"
-                        << "dash line"
-                        << "dash dot line"
-                        << "dash dot dot line";
+            lineStyles << QStringLiteral("solid line")
```

Comment:
Please undo/revert these lines as I will be blowing them away in the next 24 Hours... :smiling_imp: